README.md

```diff
@@ -1,112 +1,327 @@
 IRIDA Import Tool for Galaxy
 ============================
 
-This is a Galaxy tool that imports sequence data from IRIDA to Galaxy.
+This is a [Galaxy][galaxy] tool that imports sequence data from [IRIDA][irida] to Galaxy. Data is imported as a series of links (instead of directly copying the files from IRIDA to Galaxy).
+This requires that the IRIDA and Galaxy instance both share the same filesystem.
 
+* [1. Usage](#1-usage)
```

Comment:
In case someone was interested, I was using [md_toc](https://github.com/frnmst/md-toc) to generate this.